pkg/asset/installconfig/ovirt/config.go

```diff
@@ -85,3 +88,16 @@ func (c *Config) Save() error {
 	}
 	return ioutil.WriteFile(path, out, 0600)
 }
+
+// getValidatedConnection will create a connection and validate it before returning.
+func (c *Config) getValidatedConnection() (*ovirtsdk.Connection, error) {
+	connection, err := getConnection(*c)
+	if err != nil {
+		return nil, fmt.Errorf("failed to build configuration for ovirt connection validation (%v)", err)
```

Comment:
Use `Wrap` from "github.com/pkg/errors".
```suggestion
		return nil, errors.Wrap(err, "failed to build configuration for ovirt connection validation)
```